src/mc_bench/migrations/versions/688482827ae4_change_openai_and_gemini_provider_class.py

```diff
@@ -0,0 +1,36 @@
+"""change OpenAI and Gemini provider class
+
+Revision ID: 688482827ae4
+Revises: 9037d32770e9
+Create Date: 2024-12-13 18:11:51.318675
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '688482827ae4'
+down_revision: Union[str, None] = '9037d32770e9'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.execute("""\
+        INSERT INTO specification.provider_class (created_by, name) VALUES (
+            (select id from auth.user where username = 'SYSTEM'),
+            'OPENAI_SDK'
+        ) ON CONFLICT (name) DO NOTHING;
+               
+        INSERT INTO specification.provider_class (created_by, name) VALUES (
+            (select id from auth.user where username = 'SYSTEM'),
+            'GEMINI_SDK'
+        ) ON CONFLICT (name) DO NOTHING;
```

Comment:
```suggestion
        ) ON CONFLICT (name) DO NOTHING;
        
        UPDATE specification.provider
        SET provider_class = 'OPENAI_SDK'
        WHERE provider_class = 'OPEN_AI_SDK';
        
        DELETE FROM specification.provider_class 
        WHERE name = 'OPEN_AI_SDK';
```

Reply:
this will help keep our dev instance in a clean state (and it's good practice for once we go to prod to think through keeping things in a working and consistent state)

Reply:
I tested this locally and it appears that if you do NOT do this there will be a runtime error because there will be providers with OPEN_AI_SDK as their provider_class, but that will have no matching python class.

So please accept this suggestion before you merge this. Other than that - ship it!